elasticdl_preprocessing/layers/concatenate_with_offset.py

```diff
@@ -35,6 +35,13 @@ def __init__(self, offsets, axis=-1):
         self.axis = axis
 
     def call(self, inputs):
+        return (
+            self._call_with_none_offsets(inputs)
+            if self.offsets is None
+            else self._call_with_offsets(inputs)
```

Comment:
It is better not to write complex clauses in a single line.

Reply:
Good suggestion, updated.